src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
Please be sure that vhe value of each action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:
 `const ADD_BOOK = 'bookstore/books/ADD_BOOK'`
 `const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK'`